projects/packages/forms/src/contact-form/class-contact-form-field.php

```diff
@@ -350,6 +352,10 @@ public function render() {
 			$this->label_styles .= 'line-height: ' . (int) $this->get_attribute( 'labellineheight' ) . ';';
 		}
 
+		if ( ! empty( $block_css ) ) {
+			$this->block_styles .= esc_attr( $block_css );
```

Comment:
I think the code should just escape the entire `block_styles` property as late as possible, rather than escaping every single property that's concatenated to it (see the code that precedes this line).

I notice line 352 there's no escaping.

I think it's best tackled in a separate PR to keep the changes here as small as possible.